configuration/coding_plans.py

```diff
@@ -298,7 +298,8 @@ def get_demog_coding_plans(pipeline_name):
                            cleaner=somali.DemographicCleaner.clean_gender,
                            coded_field="gender_coded",
                            analysis_file_key="gender",
-                           fold_strategy=FoldStrategies.assert_label_ids_equal
+                           fold_strategy=FoldStrategies.assert_label_ids_equal,
+                           theme_distribution=Codes.TRUE
```

Comment:
```suggestion
                           include_in_theme_distribution=true
```
 - saying 'include_in', or similar, makes it clearer what this is for
 - This is just configuration rather than a coding decision, so I think we can just use Python's built-in booleans rather than code strings?

Reply:
updated theme_distribution -> include_in_theme_distribution.
we are less likely to make mistakes while using the imports (Codes.TRUE and Codes.FALSE) so will leave this as is.